src/olympia/accounts/tests/test_views.py

```diff
@@ -994,3 +1003,40 @@ def test_basic(self):
 
         with self.assertRaises(NotImplementedError):
             self.client.get(self.url)
+
+
+class TestSessionView(TestCase):
+    def login_user(self, user):
+        identity = {
+            'username': user.username,
+            'email': user.email,
+            'uid': user.fxa_id,
+        }
+        self.initialize_session({'fxa_state': 'myfxastate'})
+        with mock.patch(
+                'olympia.accounts.views.verify.fxa_identify',
+                lambda code, config: identity):
+            response = self.client.get(
+                '{url}?code={code}&state={state}'.format(
+                    url=reverse('accounts.authenticate'),
+                    state='myfxastate',
+                    code='thecode'))
+            cookie = response.cookies[views.JWT_TOKEN_COOKIE].value
+            assert cookie
+            verify = VerifyJSONWebTokenSerializer().validate({'token': cookie})
+            assert verify['user'] == user
+            assert self.client.session['_auth_user_id'] == str(user.id)
+            return cookie
+
+    def test_delete_when_authenticated(self):
+        user = user_factory(fxa_id='123123412')
+        token = self.login_user(user)
+        authorization = 'Bearer {token}'.format(token=token)
```

Comment:
There are two versions of the JWT, it's a bit weird.

There's one version where we provide a developer with an API key and a secret and that gets used to upload new add-ons (sent with `JWT {token}`) and another that the frontend is using where the JWT is signed by the server and it is sent with `Bearer {token}`. It's the latter that this is a log out API for.

If you wanted to log out your developer API key you could revoke it.